docs/samples/Microsoft.ML.Samples/Microsoft.ML.Samples.csproj

```diff
@@ -35,4 +35,23 @@
     
   </ItemGroup>
 
+  <ItemGroup>
+    <None Update="Dynamic\Trainers\Regression\FastTreeTemplate.tt">
+      <Generator>TextTemplatingFileGenerator</Generator>
+      <LastGenOutput>FastTreeTemplate.cs</LastGenOutput>
+    </None>
+  </ItemGroup>
+
+  <ItemGroup>
+    <Service Include="{508349b6-6b84-4df5-91f0-309beebad82d}" />
+  </ItemGroup>
```

Comment:
what is this for?

Reply:
Not sure! I added .tt files and VS studio did it automatically. Maybe @eerhardt  can tell us.